app/resources/v1/cta_resource.rb

```diff
@@ -0,0 +1,33 @@
+module V1
+  class CTAResource < JSONAPI::Resource
+    attributes :title, :description, :free, :start_time, :end_time, :cta_type, :website
+
+    relationship :location, to: :one
+    relationship :contact, to: :one
+    relationship :call_script, to: :one
+
+    filter :upcoming, apply: -> (records, value, _options) {
+      records.upcoming if value[0] == "true"
+    }
+
+    filter :cta_type, apply: ->(records, value, _options) {
+      records.where(action_type: Event::CTA_TYPES[value[0].to_sym])
+    }
+
+    def start_time
+      @model.start_at.to_i
+    end
+
+    def start_time=(start_at)
+      @model.start_at = DateTime.strptime(start_at, "%s")
```

Comment:
are we expecting to receive datetime inputs in number of seconds? My assumption would that we'd be receiving ISO8601 strings? Ditto for the getters. Do we have any specifications as to what the standards for time formatting should be? Honest question.